Tests/js/views/services/assets/ez-searchviewservice-tests.js

```diff
@@ -78,6 +78,49 @@ YUI.add('ez-searchviewservice-tests', function (Y) {
             );
            },
 
+        "Should NOT get the view parameters if no searchString provided (Regression test from EZP-26570)": function () {
+            var params;
+
+            this.request = {
+                params: {
+                    searchString: '',
+                    limit: this.limit
+                }
+            };
+            this.service = new Y.eZ.SearchViewService({
+                request: this.request,
+                app: this.app,
+                loadMoreAddingNumber: this.loadMoreAddingNumber,
+                searchString : {},
+                searchResultList: {},
+                searchResultCount: {},
+                limit: {}
+            });
+
+            this.service.load(function(){});
```

Comment:
you don't really test that the callback is called since you had this exact issue while writing the patch, it's probably something important to test :)